src/EventSubscriber/TideSiteRouteAlter.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace Drupal\tide_site\EventSubscriber;
+
+use Drupal\Core\Routing\RouteSubscriberBase;
+use Symfony\Component\Routing\RouteCollection;
+
+/**
+ * Class TideSiteRedirection.
+ *
+ * @package Drupal\tide_site\RouteSubscriber
+ */
+class TideSiteRouteAlter extends RouteSubscriberBase {
+
+  /**
+   * Alter system.admin_content route to the view '/summary_contents'.
+   *
+   * {@inheritDoc}.
+   */
+  protected function alterRoutes(RouteCollection $collection) {
+    if ($route = $collection->get('system.admin_content')) {
+      $route->setPath('/summary_contents');
```

Comment:
hi, @sonnykt summary_media page is attached to `summary_contents` page as a tab.
i cannot say that it is not correct. `$route->setDefault('view_id', 'summary_contents')->setDefault('display_id', 'page');`
 and `$route->setPath('/summary_contents');` they are equivalent.
but we can change it to `$route->setDefault('view_id', 'summary_contents')->setDefault('display_id', 'page');`

Reply:
@vincent-gao I expect that when a user visits `/admin/content`, they stay on the same path but see the `summary_contents` view instead of `system.admin_content`. All remaining tasks or links of that page should remains.

Reply:
Hi, @sonnykt I am assured that they actually don't want existing tabs or links. what they want is just a content list page with moderation status information. they don't want files, or bulk deleting action buttons etc..

Reply:
@vincent-gao @anthony-malkoun from my understanding the wireframe is just a reference. Also, it shows 2 tabs: Pages and Media.

The current UI has more tabs and local tasks:
![image](https://user-images.githubusercontent.com/167788/62030461-7998be00-b228-11e9-813d-1d13a507145d.png)

Reply:
Alright, I will check Jess with this issue tmr. Thanks.

Reply:
Hi @sonnykt  good morning. after checking with Jess, we keep `Media` and `Import Demo Content` . basically they don't want to use or see `Conent` tab. 
We probably can use UI settings to do that. Thanks @sonnykt